script/ci.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+bazel build //...
+bazel test //...
```

Comment:
also add script/check_style

Reply:
it is a simple and small repo,  not need to have two CI jobs.  Just one `presubmit` job checking all `code-style` `build`,  and `test`.

Reply:
Sure, but one benefit of having two separate jobs is a better developer experience because it's easy to tell whether checkstyle failed or the test failed without looking into the logs.
It's actually quite easy to define two Prow jobs if we already defined one.

Reply:
One concern is that each new job takes up some extra resources on our build GKE cluster (2 GB ram IIRC). Easier to keep everything in one place to reduce flakiness